util/pkg/vfs/tests/gsfs_test.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tests
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/storage/v1"
+	"k8s.io/kops/cloudmock/gce"
+	"k8s.io/kops/upup/pkg/fi/cloudup/terraform"
+	"k8s.io/kops/util/pkg/vfs"
+)
+
+var credsFile = "./mock_gcp_credentials.json"
+
+func TestGSRenderTerraform(t *testing.T) {
+	creds, err := filepath.Abs(credsFile)
+	if err != nil {
+		t.Fatalf("failed to prepare mock gcp credentials: %v", err)
+		t.FailNow()
```

Comment:
Nit: I don't think you need FailNow with Fatalf.  Fatalf == Logf + FailNow.

https://pkg.go.dev/testing#B.Fatalf